example/components/SlidingTabNavigationExample.js

```diff
@@ -17,6 +19,18 @@ export default class SlidingTabNavigationExample extends Component {
     },
   }
 
+  _goToFirstTab = () => {
+    this.props.navigation.performAction(({ tabs, stacks }) => {
+      tabs('sliding-tab-navigation').jumpToTab('first');
+    });
+  };
+
+  _goToSecondTab = () => {
+    this.props.navigation.performAction(({ tabs, stacks }) => {
+      tabs('sliding-tab-navigation').jumpToTab('second');
```

Comment:
Classic problem with backwards compatability I guess one could just use the sliding tab view today as the classic tab bar and recommend that mean while. I will give it a try at least. Thanks for comments @satya164